app/livechat/server/methods/getFirstRoomMessage.js

```diff
@@ -0,0 +1,17 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import { Rooms, Messages } from '../../../models';
+
+Meteor.methods({
+	'livechat:getFirstRoomMessage'({ rid }) {
```

Comment:
please add some permission check.. at least for logged in users.

Reply:
@sampaiodiego, done.

Thanks.